Auth: Support google OIDC and group fetching (#70140)
grafana/grafana: docs/sources/setup-grafana/configure-security/configure-authentication/google/index.md

# ..
skip_org_role_sync = true
```

### Configure team sync for Google OAuth

> > Available in Grafana v10.1.0 and later versions.

Reviewer: Because we version our docs, we don't need to add this kind of note. You can remove it.

Author: Hey @chri2547 I'm ok with removing but we end up getting PRs to re-add as users are used to using the latest version of the documentation. Could you provide further guidance?

Example: https://github.com/grafana/grafana/pull/69419

Reviewer: Hi @Jguer , I checked with the team because you made a good point that I hadn't considered. Our guidance is to keep these kinds of notes in, and when they get stale (for example, 9.5 docs refer to 7.x), remove them as we come across them.

> > Available in Grafana v10.1.0 and later versions.

Reviewer: ```suggestion
```

> Team sync allows you to automatically add users to teams based on their Google groups. The following example shows how to configure team sync for Google OAuth.

Reviewer: ```suggestion
With team sync, you can easily add users to teams by utilizing their Google groups. To set up team sync for Google OAuth, refer to the following example.
```

> 1. Enable Cloud Identity API on your [organization's dashboard](https://console.cloud.google.com/apis/api/cloudidentity.googleapis.com/)

Reviewer: ```suggestion
1. Enable the Google Cloud Identity API on your [organization's dashboard](https://console.cloud.google.com/apis/api/cloudidentity.googleapis.com/).
```

> 2. Add the `https://www.googleapis.com/auth/cloud-identity.groups.readonly` scope to your Grafana `[auth.google]` configuration:

Reviewer: ```suggestion
1. Add the `https://www.googleapis.com/auth/cloud-identity.groups.readonly` scope to your Grafana `[auth.google]` configuration:
```

> Example:

Reviewer: indent

```ini
[auth.google]
# ..
scopes = openid email profile https://www.googleapis.com/auth/cloud-identity.groups.readonly

> ```

Reviewer: indent

> 3. Configure team sync in your Grafana team's `External group sync` tab.

Reviewer: ```suggestion
1. Configure team sync in your Grafana team's `External group sync` tab.
```

>    The external group ID for a Google group is the group's email address, such as `dev@grafana.com`

Reviewer: ```suggestion
   The external group ID for a Google group is the group's email address, such as `dev@grafana.com`.
```
